### PR TITLE
added marker points for both human and mouse, updated mouse parameter…

### PR DESCRIPTION
**Pull request by arti-sukasem**
…s, updated test_lung

### CODE REVIEW

File: src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py [hunk 1]
```diff
@@ -343,67 +343,119 @@ def generateBaseMesh(cls, region, options):
...
+            for marker in markerList:
+                annotationGroup = marker["group"]
+                markerPoint = markerPoints.createNode(nodeIdentifier, markerTemplateInternal)
+                cache.setNode(markerPoint)
+                markerLocation.assignMeshLocation(cache, mesh.findElementByIdentifier(marker["elementId"]),
+                                                  marker["xi"])
+                markerName.assignString(cache, annotationGroup.getName())
+                annotationGroup.getNodesetGroup(nodes).addNode(markerPoint)
+                lungNodesetGroup.addNode(markerPoint)
+                nodeIdentifier += 1
```
Comment (rchristie) [inline src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py:396]: This code is common for all species: move to end.

File: src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py [hunk 2]
```diff
@@ -343,67 +343,119 @@ def generateBaseMesh(cls, region, options):
...
                 lowerRightNodeIds, upperRightNodeIds, elementIdentifier)
 
+            # Marker points
+            lungNodesetGroup = lungGroup.getNodesetGroup(nodes)
+            markerList = []
+
+            lowerLeftElementCount = (lElementsCount1 * (lElementsCount2-1) * lElementsCount3 + lElementsCount1)
+
+            leftApexGroup = findOrCreateAnnotationGroupForTerm(annotationGroups, region,
+                                                               get_lung_term("apex of left lung"))
```
Comment (rchristie) [inline src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py:353]: I would like all annotation groups common to all species made outside of the species-specific code. Only groups specific to one species should be made within the species-specific code, but even then I would like the annotation groups all made together at the beginning of the section.

**Review by rchristie — CHANGES_REQUESTED** [review]
Mostly have some issues with the point placement and naming - read comments. With some changes made I'll review again. Comments are in the code.

File: src/scaffoldmaker/annotation/lung_terms.py [hunk 3]
```diff
@@ -4,18 +4,29 @@
 
 # convention: preferred name, preferred id, followed by any other ids and alternative names
 lung_terms = [
+    ("apex of accessory lung", "None"),
+    ("apex of left lung", "None"),
+    ("apex of right lung", "None"),
+    ("dorsal base of accessory lung", "None"),
```
[the remark this answers is not in the document]
Comment (arti-sukasem) [reply src/scaffoldmaker/annotation/lung_terms.py:10]: FYI, its most dorsal extent on the base is Node 129. Would you still like to stick with Node 130?

File: src/scaffoldmaker/annotation/lung_terms.py [hunk 4]
```diff
@@ -4,18 +4,29 @@
...
+    ("dorsal base of left lung", "None"),
+    ("dorsal base of right lung", "None"),
```
[the remark this answers is not in the document]
Comment (arti-sukasem) [reply src/scaffoldmaker/annotation/lung_terms.py:12]: one for both right and left lungs? or add one into each lung?

File: src/scaffoldmaker/annotation/lung_terms.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (rchristie) [reply src/scaffoldmaker/annotation/lung_terms.py:10]: I don't consider 129 and 125 to be on the base (diaphragm) any more as that region is raised higher.

File: src/scaffoldmaker/annotation/lung_terms.py [hunk 5]
```diff
@@ -4,18 +4,29 @@
 
 # convention: preferred name, preferred id, followed by any other ids and alternative names
 lung_terms = [
+    ("apex of accessory lung", "None"),
+    ("apex of left lung", "None"),
+    ("apex of right lung", "None"),
```
[the remark this answers is not in the document]
Comment (rchristie) [reply src/scaffoldmaker/annotation/lung_terms.py:9]: No apex of accessory lobe is fine at 165. It's the apex of right lung that needs to move to where node 124 is.

File: src/scaffoldmaker/annotation/lung_terms.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (rchristie) [reply src/scaffoldmaker/annotation/lung_terms.py:12]: Ideally both; are you happy to edit the base of the human lung to make a point at the dorsal tip?

File: src/scaffoldmaker/annotation/lung_terms.py [hunk 6]
```diff
@@ -4,18 +4,29 @@
...
+    ("dorsal base of accessory lung", "None"),
+    ("dorsal base of left lung", "None"),
+    ("dorsal base of right lung", "None"),
     ("horizontal fissure of right lung", "None"),
-    ("lung", "UBERON:0002048", "ILX:0726937"),
+    ("lateral side of right lung", "None"),
```
[the remark this answers is not in the document]
Comment (rchristie) [reply src/scaffoldmaker/annotation/lung_terms.py:14]: FYI please make sure the element the marker is located in is the one in the middle lobe, not any of the other elements currently meeting at that point.

File: src/scaffoldmaker/annotation/lung_terms.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (arti-sukasem) [reply src/scaffoldmaker/annotation/lung_terms.py:12]: Yes, no problem Richard. If they are useful and you think it's worth having them, I will add them

**Review by rchristie — APPROVED** [review]

**Pull request merged**